src/Skeleton/PostCreateProject.php

```diff
@@ -376,6 +376,44 @@ private static function reconfigureApplication(Event $event): void
         );
         file_put_contents($projectDir . '/config/packages/twig.yaml', $content);
 
+        $io->notice('→ Reconfigure monolog');
+        $content = file_get_contents($projectDir . '/config/packages/monolog.yaml');
+
+        // Add the audit trail handler
+        $matches = [];
+        preg_match('|monolog:|', $content, $matches, PREG_OFFSET_CAPTURE);
+        $offset = mb_strpos($content, PHP_EOL, $matches[0][1]) + 1;
+        $insert = [
+            '   handlers:',
+            '       audit_trail:',
+            '           type: rotating_file',
+            '           path: "%kernel.logs_dir%/audit.log"',
+            '           level: info',
+            '           channels: [\'audit_trail\']',
+            '           max_files: 10',
```

Comment:
10 files is 10 dagen?

Reply:
is indd 10 dagen

Reply:
Doe maar 30 dagen by default.

Reply:
fixed aa9bbec86c3730f2228d2b6973dd5d9c72ee5b75